extensions/SemanticResultFormats/formats/Exhibit/SRF_Exhibit.js

```diff
@@ -102,7 +102,7 @@ function createExhibit() {//overload Exhibit's autoCreate-functionality
     for(var i = 0; i < ex_linkcounter; i++){ //lenses
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "i" as this name is already used in declaration at line 82. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2814)

[project-id]: wikia:app